apps/api-reference/src/apis/evm/get-ema-price-no-older-than.ts

```diff
@@ -0,0 +1,65 @@
+import Btc from "cryptocurrency-icons/svg/color/btc.svg";
+import Eth from "cryptocurrency-icons/svg/color/eth.svg";
```

Comment:
can we use the ones we are currently using on our website for standardization purposes?

Reply:
I think that's a good idea but because the icons are not packaged up nicely I'm going to wait until I can migrate the main website into the monorepo to avoid duplicating all the svgs and related code all over the place.

Once I can move that into the monorepo I'll extract the icons as a package and re-use it here.